libraries/lib-numeric-formats/NumericConverterRegistry.h

```diff
@@ -38,21 +38,24 @@ using NumericConverterFormatterFactoryPtr =
 // NumericConverterRegistryGroupTag is a fake type needed to fix the link on Windows
 struct NumericConverterRegistryGroupTag {};
 
-struct NUMERIC_FORMATS_API NumericConverterRegistryGroup :
-    public Registry::GroupItem<NumericConverterRegistryGroupTag>
-{
-   template <typename... Args>
-   NumericConverterRegistryGroup(
-      const Identifier& internalName, NumericConverterType _type,
-      Args&&... args)
-       : GroupItem { internalName, std::forward<Args>(args)... }
-       , type { std::move(_type) }
-   {
-   }
+struct NumericConverterRegistryGroupData {
```

Comment:
I'm not quite sure what is the benefit of having one more type, I don't have any strong opinion though